tests/connectors/kubernetes_test.py

```diff
@@ -1109,7 +1112,10 @@ async def test_adjust_tuning_insufficient_resources(
             setting_name="mem",
             value="128Gi", # impossible right?
         )
-        with pytest.raises(AdjustmentRejectedError, match="Insufficient memory.") as rejection_info:
+        with pytest.raises(
+            AdjustmentRejectedError,
+            match=re.escape("Insufficient memory. Unschedulable adjustment value: fiber-http/fiber-http-tuning.mem=128Gi (Reason unschedulable)")
```

Comment:
Nit but I'd prolly do something like `(Reason: 'unschedulable')` so it easier to parse quickly

Reply:
The reason is no longer included in the message per Peter's ticket feedback; sending reason in the message is redundant and should be logged clearly in the backend